pom.xml

```diff
@@ -138,10 +135,8 @@
           <!-- <testClassesDirectory>census-rh-cucumber/src/test/java</testClassesDirectory> -->
```

Comment:
why not remove this commented out line while you are here ?

Reply:
I think it's wrong anyway - shouldn't be RH - should be 
<testSourceDirectory>census-field-service-cucumber/src/test/java</testSourceDirectory>